docs/classes/ManifestBuilder.md

```diff
@@ -37,12 +37,12 @@
 
 | Name | Type |
 | :------ | :------ |
-| `baseDefinition` | [`BaseManifestDefinition`](../modules.md#basemanifestdefinition) |
+| `baseDefinition` | [`BaseManifestDefinition`](../README.md#basemanifestdefinition) |
 | `options?` | `ManifestBuilderOptions` |
 
 #### Defined in
 
-[lib/manifestBuilder.ts:41](https://github.com/contentauth/c2pa-node/blob/fb1d732/js-src/lib/manifestBuilder.ts#L41)
```

Comment:
We might need to not do this through a fork, since it seems to be changing the URLs to your fork. Can you push this PR to the main repo, rebuild, and open up a PR from there?

Reply:
Good catch.  Moved to a branch and created new PR.